tests/publiccloud/boottime.pm

```diff
@@ -119,6 +130,18 @@ sub run {
         ssh_access_hard => 60,
     };
 
+    my $threshold_by_flavor = {
+        qr/^(Azure-Standard|Azure-Basic)$/ => {
+            overall   => 3 * 60,
+            userspace => 3 * 60,
```

Comment:
overall should be higher than userspace, right? 
Why are you setting now overall to 180 if it was 160 in the default list?

Reply:
ok